Tasks/Common/webdeployment-common/deployusingmsdeploy.ts

```diff
@@ -78,6 +78,55 @@ export async function DeployUsingMSDeploy(webDeployPkg, webAppName, publishingPr
     }
 }
 
+function argStringToArray(argString): string[] {
```

Comment:
yes, but if the user wants to specify this additional argument with double spaces, they will have to specify it like so: --setparam:name="'<param name with consecutive double spaces>'", value="'<value with consecutive double spaces>'"